ui/pages/routes/routes.component.tsx

```diff
@@ -687,10 +685,17 @@ export default function Routes() {
       <Suspense fallback={null}>
         {/* since the loading time is less than 200ms, we decided not to show a spinner fallback or anything */}
         <Switch>
-          {/** @ts-expect-error TODO: Replace `component` prop with `element` once `react-router` is upgraded to v6 */}
-          <Route path={ONBOARDING_ROUTE} component={OnboardingFlow} />
-          {/** @ts-expect-error TODO: Replace `component` prop with `element` once `react-router` is upgraded to v6 */}
-          <Route path={LOCK_ROUTE} component={Lock} exact />
+          <RouteWithLayout
```

Comment:
In general, these Routes were just converted from
```
<Route path={...} component={Component} />
```
to
```
<RouteWithLayout path={...} component={Component} layout={LegacyLayout} />
```
to reduce boilerplate and in preparation for React Router v6